sympy/integrals/transforms.py

```diff
@@ -1167,38 +1167,56 @@ def _compute_transform(self, f, t, s, **hints):
         a = Wild('a', exclude=[t])
         b = Wild('b', exclude=[t])
         c = Wild('c', exclude=[t])
+        g = WildFunction('g', nargs=1)
         tau = Wild('tau', exclude=[t])
         omega = Wild('omega', exclude=[t])
         dco = lambda f: self._deep_collect(f,t)
-        simple_rules = [
-            # ( time domain, laplace domain,
-            #       condition, convergence plane, preparation function )
-            (exp(a*t-tau), exp(-tau)/(s-a),
-                 S.true, a, dco), # 3.1
-            (t*exp(a*t-tau), exp(-tau)/(s-a)**2,
-                 S.true, a, dco), # 3.2
-            (t**c*exp(a*t), gamma(c+1)/(s-a)**(c+1),
-                 c>-1, a, dco), # 3.3
-            (exp(-a*t**2), sqrt(pi/4/a)*exp(s**2/4/a)*erfc(s/sqrt(4*a)),
-                 a>0, S.Zero, dco), # 3.6
-            (sin(omega*t), omega/(s**2+omega**2),
-                 S.true, S.Zero, dco), #6.1
-            (sin(omega*t)/t, atan(omega/s),
-                 S.true, S.Zero, dco), # 6.6
-            (cos(omega*t), s/(s**2+omega**2),
-                 S.true, S.Zero, dco), # 6.11
-            (exp(b*t)*sin(a*t), a/((s-b)**2+a**2),
-                 S.true, b, dco), # 6.21
-            (exp(b*t)*cos(a*t), (s-b)/((s-b)**2+a**2),
-                 S.true, b, dco), # 6.22
-            ]
-        k, func = f.as_independent(t, as_Add=False)
-        for t_dom, s_dom, check, plane, prep in simple_rules:
-            ma = prep(func).match(t_dom)
-            if ma:
-                if check.xreplace(ma):
-                    return self._cr(k*s_dom.xreplace(ma), plane.xreplace(ma),
-                                    S.true, **hints)
+        if not any(func.has(t) for func in f.atoms(AppliedUndef)):
+            simple_rules = [
+                # ( time domain, laplace domain,
+                #       condition, convergence plane, preparation function )
+                (exp(a*t-tau), exp(-tau)/(s-a),
+                     S.true, a, dco), # 3.1
+                (t*exp(a*t-tau), exp(-tau)/(s-a)**2,
+                     S.true, a, dco), # 3.2
+                (t**c*exp(a*t), gamma(c+1)/(s-a)**(c+1),
+                     c>-1, a, dco), # 3.3
+                (exp(-a*t**2), sqrt(pi/4/a)*exp(s**2/4/a)*erfc(s/sqrt(4*a)),
+                     a>0, S.Zero, dco), # 3.6
+                (t*exp(-a*t**2),
+                     1/(2*a)-2/sqrt(pi)/(4*a)**(3/2)*s*erfc(s/sqrt(4*a)),
```

Comment:
Change `3/2` to `Rational(3, 2)`, `S(3)/2`, or `Integer(3)/2` to avoid floats (created by Python before SymPy gets hold of the input). (Not clear where the 0.25 in the test comes from though. Does it print like that or is it a consequence of `Eq(0.25, Rational(1, 4)) = True`?)

Reply:
> ... to avoid floats (created by Python before SymPy gets hold of the input) ...

Thanks! This single sentence solved SOOO many of my open questions ;)  Changed, and added four more rules with test cases.